tutorials/pyroot/pyroot001_arrayInterface.py

```diff
@@ -0,0 +1,46 @@
+## \file
+## \ingroup tutorial_pyroot
+## \notebook -nodraw
+## This tutorial illustrates the conversion of TVec vectors to numpy arrays
+## without copying the data.
+## The memory-adoption is achieved by the dictionary __array_interface__, which
+## is added dynamically to the Python objects by PyROOT.
+##
+## \macro_code
+##
+## \date April 2018
+## \author Stefan Wunsch
+
+import ROOT
+from sys import exit
+
+
+def main():
+    # Create a TVec ROOT object and assign values
+    tvec = ROOT.Experimental.VecOps.TVec("float")(2)
+    tvec[0] = 1
+    tvec[1] = 2
+    print("Content of the TVec ROOT object: {}".format([x for x in tvec]))
+
+    # Interface TVec with a numpy array
+    array = np.asarray(tvec)
+    print("Content of the associated numpy array: {}".format(
+        [x for x in array]))
+
+    # The numpy array adopts the memory of the TVec without copying the content.
+    # Note that the first entry of the numpy array changes when assigning a new
+    # value to the first entry of the tvec.
+    tvec[0] = 42
+    print("Content of the numpy array after changing an entry of the TVec: {}".
+          format([x for x in array]))
+
+    # Use numpy features on data of ROOT objects
+    print("Mean of the numpy array entries: {}".format(np.mean(array)))
+
+
+if __name__ == "__main__":
```

Comment:
Does it make sense to have this? Should the tutorial be flat? @etejedor

Reply:
It hides the lazy import of numpy so that the incrementals do not fail if ROOT is built/tested without numpy in the path. That was actual the idea so that this logic can be moved to the bottom of the file. But true, it does not make sense in terms of preventing undesired imports when importing this file from other python scripts.

Reply:
If possible I would do as in the other tutorials and keep the flat schema.

Reply:
Made it flat.